site/content/docs/5.0/examples/cheatsheet-rtl/index.html

```diff
@@ -1418,6 +1418,36 @@ <h3>ترقيم الصفحات</h3>
         {{< /example >}}
       </div>
     </article>
+    <article class="my-3" id="stepped-process">
+      <div class="bd-heading sticky-xl-top align-self-start mt-5 mb-3 mt-xl-0 mb-xl-2">
+        <h3>Stepped process</h3>
```

Comment:
This heading isn't translated. From Google Translate:
```suggestion
        <h3>عملية متدرجة</h3>
```